File: telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/command/status/StatusCommand.java
package com.github.bumblebee.command.status;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Component;

import com.github.bumblebee.command.SingleArgumentCommand;

import telegram.api.BotApi;
import telegram.domain.Update;

@Component
public class StatusCommand extends SingleArgumentCommand {

    private final BotApi botApi;

    @Autowired
    public StatusCommand(BotApi botApi) {

        this.botApi = botApi;
    }

    @Override
    public void handleCommand(Update update, Long chatId, String argument) {
        botApi.sendMessage(chatId, "I'm fine!");
    }

}

File: telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/command/status/StatusCommand.kt
package com.github.bumblebee.command.status

import com.github.bumblebee.command.SingleArgumentCommand
import com.github.telegram.api.BotApi
import org.springframework.stereotype.Component
import com.github.telegram.domain.Update

@Component
class StatusCommand(val botApi: BotApi) : SingleArgumentCommand() {

    override fun handleCommand(update: Update, chatId: Long, argument: String?) {
        botApi.sendMessage(chatId, "I'm fine!")
    }
}